### PR TITLE
Cypress/E2E: Add CWS option to connect with cloud test server

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add CWS option to connect with cloud test server

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress.json [hunk 1]
```diff
@@ -31,6 +33,7 @@
         "resetBeforeTest": false,
         "runLDAPSync": true,
         "secondServerURL": "http://localhost/s/p",
+        "serverEdition": "Team",
```
Comment (saturninoabril) [inline e2e/cypress.json:36]: This is not new and added here to be explicit on default. Options care found at https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/utils/constants.js#L11-L13

File: e2e/cypress/integration/modals/product_notice_spec.js [hunk 2]
```diff
@@ -1,103 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (saturninoabril) [inline e2e/cypress/integration/modals/product_notice_spec.js:1]: Not related to CWS, but remove to clean up unused spec.

**Review by neallred — APPROVED** [review]
looks good to me!

**Review by josephbaylon — APPROVED** [review]
LGTM

**Comment by saturninoabril** [comment]
I'll go ahead and merge since this is required to other follow up fix on master.

**Pull request merged**